test/utils.test.js

```diff
@@ -0,0 +1,77 @@
+// import React from 'react'
+// import { format } from 'date-fns'
+
+import { isSelectable, mergeModifiers } from '../src/utils'
+import { isToday } from 'date-fns'
+
+describe('isSelectable', () => {
+  it('should return `true` for a date that is within the range of `minimumDate` and `maximumDate`', () => {
+    const testDate = new Date(2025, 10, 4)
+    const minimumDate = new Date(2025, 10, 3)
+    const maximumDate = new Date(2025, 10, 5)
+    expect(isSelectable(testDate, { minimumDate, maximumDate })).toBe(true)
+  })
+  it('should return `true` for a date that is equal to `minimumDate` or `maximumDate`', () => {
+    const minimumDate = new Date(2025, 10, 3)
+    const maximumDate = new Date(2025, 10, 5)
+    expect(isSelectable(minimumDate, { minimumDate, maximumDate })).toBe(true)
+    expect(isSelectable(maximumDate, { minimumDate, maximumDate })).toBe(true)
+  })
+  it('should return `false` for a date that is before `minimumDate` or after `maximumDate`', () => {
+    const beforeTestDate = new Date(2025, 10, 2)
+    const afterTestDate = new Date(2025, 10, 6)
+    const minimumDate = new Date(2025, 10, 3)
+    const maximumDate = new Date(2025, 10, 5)
+
+    expect(isSelectable(beforeTestDate, { minimumDate, maximumDate })).toBe(false)
+    expect(isSelectable(afterTestDate, { minimumDate, maximumDate })).toBe(false)
+  })
+})
+describe('mergeModfiers', () => {
+  it('should return the original modifier object if `newModifiers` is not provided', () => {
+    const today = new Date(2025, 10, 4)
+    const mockBaseModifiers = { today: isToday(today) }
+    expect(mergeModifiers(mockBaseModifiers)).toEqual(mockBaseModifiers)
+  })
+
+  it('should (in effect) merge a provided modifier into the `modifiers` object if the key is not already present', () => {
+    const mockBaseModifiers = { today: (date) => false }
+    const newModifier = { thursday: (date) => true }
+
+    expect(mergeModifiers(mockBaseModifiers, newModifier)).toEqual({
+      ...mockBaseModifiers,
+      ...newModifier
+    })
+    expect(mergeModifiers(mockBaseModifiers, newModifier)).toEqual({
+      ...newModifier,
+      ...mockBaseModifiers
+    })
+  })
+
+  it('should check both `baseModifier` and `newModifier` of the same name', () => {
```

Comment:
I think a better way to test this one, without worrying about what values are passed, might be using [`toHaveBeenCalled`](https://jestjs.io/docs/en/expect#tohavebeencalled), something like:

```
const modifierA = jest.fn()
const modifierB = jest.fn()

const baseModifiers = { test: modifierA }
const newModifiers = { test: modifierB }
const mergedModfiers = mergeModifiers(baseModifiers, newModifiers)

mergedModfiers.test()

expect(modifierA).toHaveBeenCalled()
expect(modifierB).toHaveBeenCalled()
```